README.md

```diff
@@ -9,3 +9,11 @@ This repository contains codes for time series prediction by LSTM, especially wh
 2. torch == 1.6.0
 
 3. blitz (for bayesian lstm)
+
+4. Python >= 3.6
```

Comment:
FYI: This recommended Python versions referred from [blitz-bayesian-pytorch installation guide](https://pypi.org/project/blitz-bayesian-pytorch/).